backups/snapshots/2025-08-30_15-46-05/_autopolicy.py

```
@@ -0,0 +1,142 @@
+                    status = getattr(r, "status", 200); content = r.read(); content_len = len(content or b"")
+                    if 200 <= status < 300:
+                        dt = time.perf_counter() - t0
+                        if metrics: metrics.log(url, host, status, dt, attempt, content_len)
+                        return content
+                    raise RuntimeError(f"bad status {status}")
+        except Exception as e:
+            err = str(e); dt = time.perf_counter() - t0
+            if metrics: metrics.log(url, host, status or 0, dt, attempt, content_len, error=err)
+            retryable = (status in (429,500,502,503,504)) or ("timed out" in (err or "").lower())
+            if attempt < retrier.max_retries and retryable:
+                time.sleep(Retrier({"default":{"max_retries":0,"timeout_s":0}}).backoff(attempt)); continue
```

Ruff (0.12.2)

Multiple statements on one line (semicolon)

(E702)